### DISCUSSION TITLE
Advanced Access Point Settings（高级接入点设置

### DISCUSSION THREAD

**Discussion by huzesama** [Mygod/VPNHotspot#585] [Q&A]
我不清楚这个WiFi热点设置里的高级接入点是什么意思（设备白名单）。它的作用是什么？以及它是如何配置的？

**Comment by Mygod** [comment]
I believe your phone needs to support feature `CLIENT FORCE DISCONNECT` to use this feature. Basically, a device is allowed to connect only if its MAC address is in the list.

**Reply by huzesama** [reply]
“隐藏网络”上面的那一项呢？它只允许17个字符（一条MAC地址）。我输入了我的主要设备MAC，但似乎没有什么事情发生。
![Screenshot_2024-06-22-10-55-12-928_be mygod vpnhotspot](https://github.com/Mygod/VPNHotspot/assets/84225197/add51e43-a7ca-4a7f-8f4b-2460213d38bf)

**Reply by Mygod** [reply]
All the configurations are simply suggested to the Android system. It is up to your system and your driver to decide how to use it.

**Reply by huzesama** [reply]
Okay, Thank u
